### PR TITLE
Cite paper on bitwise analysis

### LINKED ISSUE #47: New analysis
New analysis: https://link.springer.com/chapter/10.1007/978-981-97-7737-2_1

I got a copy of the paper. To summarize:

- With a 256 bit tag (they quote the construction used in the draft), AEGIS-256 could claim 256 bit security against forgery.
- But with AEGIS-128L and a 256 bit tag, probably of a successful forgery is 2^-216, so > 2^-256.

The fact that a state collision on AEGIS-128L has a complexity above 2^128 but below 2^256 isn't new. This is why in the security guarantees section, we only guarantee 128 bit security regardless of the variant. 

What's new is the fact that we *could* claim 256 bit security against forgeries for AEGIS-256.

I don't know if we should, or if we should remain conservative as this has little practical implications and further analysis may invalidate that claim.

Regardless, that's one more paper we can cite.

Screenshot 2024-09-19 at 01 12 18
Screenshot 2024-09-19 at 01 14 04

### PR DESCRIPTION
**Pull request by jedisct1**
Fixes #47

### CODE REVIEW

File: draft-irtf-cfrg-aegis-aead.md [hunk 1]
```diff
@@ -226,6 +226,26 @@ informative:
...
+      DOI: 10.1007/978-981-97-7737-2_1
+    author:
+      -
+        ins: T. Shiraya
+        name: Takuro Shiraya
+        org: University of Hyogo
+      -
+        ins: K. Sakamoto
+        name: Kosei Sakamoto
+        org: University of Hyogo
```
Comment (samuel-lucas6) [inline draft-irtf-cfrg-aegis-aead.md]: Should be `Mitsubishi Electric Corporation`.

File: draft-irtf-cfrg-aegis-aead.md [hunk 2]
```diff
@@ -226,6 +226,26 @@ informative:
...
+      -
+        ins: T. Isobe
+        name: Takanori Isobe
+        org: University of Hyogo; National Institute of Information and Communications Technology
```
Comment (samuel-lucas6) [inline draft-irtf-cfrg-aegis-aead.md]: `National Institute of Information and Communications Technology` isn't mentioned in this paper so not sure if he's still there.

**Review by samuel-lucas6 — CHANGES_REQUESTED** [review]
Two author changes. Otherwise, looks good and happy with the `at least 128-bit security` claim. Probably safest for now.

**Comment by jedisct1** [comment]
Shall we also update their affiliations in other cited papers?

**Comment by samuel-lucas6** [comment]
> Shall we also update their affiliations in other cited papers?

I think we should just copy each paper since it's correct at the time of publication. It'll get confusing otherwise.

**Pull request merged**